code/ui/scoreboard/Scoreboard.razor

```diff
@@ -0,0 +1,145 @@
+﻿@using System.Collections.Generic;
+
+@namespace Cinema.UI
+
+<style>
```

Comment:
I recommend that you split the css to a .scss file

Reply:
Sounds good, I moved the styling out of the razor file to a .scss file in the newest commit.